### PR TITLE
Pat/removed downloaded file

### PR DESCRIPTION
**Pull request by whlpatricia**
### Description

Files created from the tests are now removed so nothing is left after test conclusion.

### Bugzilla bug ID

**Link:** 
https://bugzilla.mozilla.org/show_bug.cgi?id=1918759
https://bugzilla.mozilla.org/show_bug.cgi?id=1918530

### Type of change

- [x] Other Changes (Fixed defect)

### How does this resolve / make progress on that bug?

Completed

### Screenshots / Explanations

N/A

### Comments / Concerns

N/A

### CODE REVIEW

File: tests/downloads/test_mixed_content_download_via_https.py [hunk 1]
```diff
@@ -13,11 +13,14 @@
 def test_case():
     return "1756722"
 
+@pytest.fixture()
+def delete_files_regex_string():
+    return r"download"
```
Comment (ben-c-at-moz) [inline tests/downloads/test_mixed_content_download_via_https.py:18]: Can we write a more specific regex? Don't want to accidentally blast a bunch of unrelated downloads.

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]

File: tests/downloads/test_mixed_content_download_via_https.py (hunk 1 above)
Comment (whlpatricia) [reply tests/downloads/test_mixed_content_download_via_https.py:18]: wouldwould '\bdownload\b' be good?

Comment (ben-c-at-moz) [reply tests/downloads/test_mixed_content_download_via_https.py:18]: It would be better, I think

File: tests/downloads/test_add_zip_type.py [hunk 2]
```diff
@@ -28,7 +34,11 @@ def test_add_zip_type(driver: Firefox):
 
     # Click on the available zip
     web_page.find_element(
-        By.XPATH, "//td/a[@href='/pub/firefox/releases/0.9rc/Firefox-win32-0.9rc.zip']"
+        By.XPATH, '//*[@id=":R55ab:"]'
```
Comment (ben-c-at-moz) [inline tests/downloads/test_add_zip_type.py:37]: The test works, but let's talk about selector strategies soon

**Review by ben-c-at-moz — CHANGES_REQUESTED** [review]
Let's talk about selector strategies & using the BasePage, and revisit / revise a tiny bit of the test. The fix itself is super good!

**Review by ben-c-at-moz — APPROVED** [review]
Looks great, ready to merge!

**Pull request merged**